.github/workflows/integration.yaml

```diff
@@ -103,6 +103,7 @@ jobs:
           TEST_LXD_IMAGE: ${{ matrix.os }}
           TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
         run: |
+          python --version
```

Comment:
running pytest through `tox -e integration` should be printing the version right?